src/gui/updater/appimageupdater.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (C) 2022 by Fabian Müller <fmueller@owncloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#include <QMessageBox>
+#include <QSettings>
+#include <QTimer>
+#include <appimage/update.h>
+
+#include "appimageupdater.h"
+#include "common/version.h"
+#include "theme.h"
+#include "updater_private.h"
+// SettingsDialog is apparently forward-declared somewhere, need to include the header to make the compilation work
```

Comment:
that comment is a bit weird, especially as it comes before the wrong include.

Reply:
`clang-format` apparently reordered the includes here and did not keep the comment with the source. I'll just dump the comment.